sdk/src/main/java/com/meniga/sdk/eventconverters/generic/MenigaTransactionEventConverter.java

```diff
@@ -36,7 +36,6 @@ public MenigaEvent eventConverter(JsonElement element) {
 		List<String> types = new ArrayList<>();
 		types.add("transactions_threshold_deposit");
 		types.add("transactions_category_budget_watch");
-		types.add("transactions_merchant_count");
```

Comment:
Is there a reason for removing this transaction event type?

Reply:
It is already defined in https://github.com/meniga/mobile-sdk-android/blob/c809022f1cd616df4f7931a366098b0961e0a144/sdk/src/main/java/com/meniga/sdk/eventconverters/generic/MenigaTransactionCountEventConverter.java#L39 and we had tests for that https://github.com/meniga/mobile-sdk-android/blob/c809022f1cd616df4f7931a366098b0961e0a144/sdk/src/test/java/com/meniga/sdk/models/feed/MenigaFeedTest.java#L75. Should those event types be declared in both classes?